src/main/scala/com/thenewmotion/chargenetwork/ocpp/charger/BosService.scala

```diff
@@ -109,7 +129,34 @@ class ConnectorServiceImpl(protected val service: SyncCentralSystem, connectorId
     service(MeterValuesReq(ConnectorScope(connectorId), Some(transactionId), List(meter)))
   }
 
+  def signedAblMeterValue(transactionId: Int, meterValue: Int): Unit = {
+    service(MeterValuesReq(ConnectorScope(connectorId), Some(transactionId), List(signedMeterValuesBeginTransaction(meterValue))))
+  }
+
+  def signedMeterValuesBeginTransaction(meter: Int): Meter = {
+    val meterValue: Value = Value(value = meter.toString,
+      context = ReadingContext.TransactionBegin,
+      format = ValueFormat.Raw,
+      measurand = Measurand.EnergyActiveImportRegister,
+      phase = None,
+      location = Location.Outlet,
+      unit = UnitOfMeasure.Kwh)
```

Comment:
why not Wh?

Reply:
Fixed

Reply:
Sorry, in fact default unit is UnitOfMeasure.W

Reply:
@akoshterek checking on the code, I see the default unit used for every meter value on simulator is `KW`: 
```scala
object DefaultValue {
  val readingContext = ReadingContext.SamplePeriodic
  val format = ValueFormat.Raw
  val measurand = Measurand.EnergyActiveImportRegister
  val location = Location.Outlet
  val unitOfMeasure = UnitOfMeasure.Wh
  val phase = None

  def apply(value: Int): Value = Value(value.toString, readingContext, format, measurand, phase, location, unitOfMeasure)

  def unapply(x: Value): Option[Int] = PartialFunction.condOpt(x) {
    case Value(value, `readingContext`, `format`, `measurand`, `phase`, `location`, `unitOfMeasure`) => value.toFloat.round
  }
}
```

Reply:
KW is also fine, thanks